crates/gpui/src/interactive.rs

```diff
@@ -55,6 +55,8 @@ impl KeyEvent for KeyUpEvent {}
 pub struct ModifiersChangedEvent {
     /// The new state of the modifier keys
     pub modifiers: Modifiers,
+    /// The new state of the capslock key
+    pub capslock: Capslock,
```

Comment:
This should be added to `modifiers`, right? I don't see why it should be an adjacent data structure :)

Reply:
Oh I see the intent now, that makes a lot of sense. I’ll retract this feedback.